### ISSUE TITLE
equalizer?

### ISSUE THREAD

**Issue by markdancestango** [fooyin/fooyin#415]
Hi I am not sure whether the feature is in fooyin already but I don't seem to think I can locate an equalizer?

Thanks

**Comment by AdamAnon** [comment]
No, there is no EQ in current build of fooyin,. If you look at the roadmap, there is mention of DSP Plugins TBD so maybe in a future release.

https://github.com/fooyin/fooyin/blob/master/ROADMAP.md

Have you tried  Easy Effects (Pipewire) or Pulse Effects (Pulse Audio), depending on your distro's audio subsystem. It gives you systemwide EQ and lots of other sound effects, bass enhancer, crystalizer, etc.

![Screenshot_20241215_140557](https://github.com/user-attachments/assets/3699c0b8-7878-4b80-a035-06554febae4b)
